operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys_on_PIT_Node.md

```diff
@@ -33,23 +36,31 @@ The Kubernetes image is used by the master and worker nodes.
 
 1. Open the image.
 
-   The Kubernetes image will be of the form "kubernetes-0.1.69.squashfs" in /var/www/ephemeral/data/k8s, but the version number may be different.
+   The Kubernetes image will be of the form `kubernetes-0.1.69.squashfs` in `/var/www/ephemeral/data/k8s`, but the version number may be different.
 
    ```bash
    pit# unsquashfs kubernetes-0.1.69.squashfs
    ```
 
+1. Remove default SSH keys
+
+   ```bash
+   pit# rm -rf squashfs-root/root/.ssh
+   pit# rm -f /etc/ssh/*key*
+   ```
+
 1. Copy the generated public and private SSH keys for the root account into the image.
 
    This example assumes that an RSA key was generated.
 
    ```bash
+   pit# mkdir -m 0700 squashfs-root/root/.ssh
    pit# cp -p /root/.ssh/id_rsa /root/.ssh/id_rsa.pub squashfs-root/root/.ssh
    ```
 
 1. Add the public SSH key for the root account to `authorized_keys`.
 
-   This example assumes that an RSA key was generated so it adds the id_rsa.pub file to authorized_keys.
+   This example assumes that an RSA key was generated so it adds the `id_rsa.pub` file to `authorized_keys`.
```

Comment:
The append to authorized keys in step # 5 should just be a write. Same with Ceph.